src/UIComponents/ModelCard.jsx

```diff
@@ -0,0 +1,156 @@
+/* React component to handle displaying the model card. */
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { styles } from "../constants";
+import { getSummaryStat,
```

Comment:
nit: our current styling would also have this first entry on a new line.  We should be consistent.